tests/components/integration.test.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, useState, xml } from "@odoo/owl";
+import { Model, Spreadsheet } from "../../src";
+import { SpreadsheetChildEnv } from "../../src/types";
+import { createSheet, setCellContent } from "../test_helpers/commands_helpers";
+import { getChildFromComponent, mountComponent, nextTick } from "../test_helpers/helpers";
+
+class Parent extends Component<{}, SpreadsheetChildEnv> {
+  static template = xml/* xml */ `
+      <div class="integration-wrapper">
+        <Spreadsheet model="model"/>
+      </div>
+    `;
+  static components = { Spreadsheet };
+
+  state = useState({ model: new Model() });
+
+  setup() {}
+
+  get model() {
+    return this.state.model;
+  }
+
+  setModel(model: Model) {
+    this.state.model = model;
+  }
```

Comment:
The test is a whole 7 line shorter if you do'nt bother to create a getter/setter, and just modify parent.state.model directly. Just sayin' :wink: